app-modules/service-management/src/Observers/ServiceRequestObserver.php

```diff
@@ -68,6 +70,19 @@ public function created(ServiceRequest $serviceRequest): void
         if ($serviceRequest->status?->classification === SystemServiceRequestClassification::Open) {
             $serviceRequest->respondent->notify(new SendEducatableServiceRequestOpenedNotification($serviceRequest));
         }
+
+        if ($serviceRequest?->priority->type->assignment_type == ServiceRequestTypeAssignmentTypes::Individual) {
+            $manager = $serviceRequest?->priority->type?->assignmentTypeIndividual;
+
+            if ($manager) {
+                $serviceRequest->assignments()->create([
+                    'user_id' => $manager->getKey(),
+                    'assigned_by_id' => auth()->user()?->id ?? null,
```

Comment:
Let's make sure to use `getKey()` here and I don't think you need the null coalescing as the `?` on `user()` should return `null` if there is no user